mutalyzer/website/views.py

```diff
@@ -673,28 +699,80 @@ def check_position(position, field):
 def description_extractor():
     """
     The Variant Description Extractor (experimental service).
-    """
-    reference_sequence = request.args.get('reference_sequence')
-    variant_sequence = request.args.get('variant_sequence')
 
-    if not (reference_sequence and variant_sequence):
-        return render_template('description-extractor.html')
+    There are two ways for the user to provide two sequences, corresponding to
+    the values for the `method` field, each requiring some additional fields to
+    be defined:
+
+    `method=raw_method`
+      The reference and sample sequences are pasted into the form fields.
+
+      - `reference_sequence`: The reference sequence.
+      - `variant_sequence`: The sample sequence.
+
+    `method=refseq_method`
+      The reference and sample sequences are given by RefSeq accession numbers.
 
+      - `reference_accession_number`: RefSeq accession number for the reference
+        sequence.
+      - `sample_accession_number`: RefSeq accession number for the sample
+        sequence.
+    """
     output = Output(__file__)
     output.addMessage(__file__, -1, 'INFO',
                       'Received Description Extract request from %s'
                       % request.remote_addr)
 
+    (r, s, reference_sequence, variant_sequence, reference_accession_number,
+        sample_accession_number, description, raw_vars) = ('', '', '', '', '',
+        '', '', None)
```

Comment:
Since strings are immutable, Instead of

    a, b, c = '', '', ''

it's safe to just write

    a = b = c = ''

Personally I think the code would be clearer if setting all the request arguments variables was done at the top:

    method = request.args.get('method')
    reference_accession_number = request.args.get('reference_accession_number', '')
    reference_sequence = request.args.get('reference_sequence', '')
    # etc

    if method == 'refseq_method':
        # etc

It would make the data flow clearer. (Indeed, I wouldn't say the reference loader view is a good example of clear code.)